src/sentry/runner/initializer.py

```diff
@@ -188,14 +188,15 @@ def configure_structlog():
 
     structlog.configure(**kwargs)
 
-    lvl = os.environ.get('SENTRY_LOG_LEVEL', None)
+    lvl = os.environ.get('SENTRY_LOG_LEVEL', 'INFO')
     if lvl in logging._levelNames:
```

Comment:
Per previous comment, we should be able to remove this check since `SENTRY_LOG_LEVEL` should be gated by `click` making sure it's a valid level.